src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -196,26 +241,13 @@ static inline int MPIDI_OFI_win_init(MPI_Aint length,
                                     NULL), openct);     /* Context: counter events   */
         MPIDI_OFI_WIN(win).issued_cntr = &MPIDI_OFI_WIN(win).issued_cntr_v;
```

Comment:
I think the `issued_cntr ` setting is stx-specific, thus it might be better if we put it in the `win_init_stx` subroutine.

Reply:
This makes some sense. If we are to move this line in `win_init_stx`, would it also make more sense to move counter creation & initialization lines as well?

CC: @shawnccx as he is preparing another patch on top of this patch.

Reply:
My PR has a stand-alone function `MPIDI_OFI_win_set_per_win_sync` if OFI-level cntr is created for per-window sync (used for shared transmit context and scalable endpoint).

Reply:
So... would it make more sense to introduce `MPIDI_OFI_win_set_per_win_sync` function here?

Reply:
Well, then probably you should merge my commit `ch4/ofi: Refactor shared transmit context setup for RMA` here.

Reply:
This makes sense to me. Or we can push the other commits for this PR and move the last one into @shawnccx 's PR. The latter might be better if shawnccx's PR introduces a complete functionality.

Reply:
> Or we can push the other commits for this PR and move the last one into @shawnccx 's PR.

I'd +1 for this approach. I'll coordinate with @shawnccx so his upcoming PR will also include reformatted "CH4/OFI: Refactor window setup" patch. I'll update this PR with a fix for the other comment, then drop the last patch from this PR.